### DISCUSSION TITLE
Unable to use custom date picker

### DISCUSSION THREAD

**Discussion by inaminute** [nick-keller/react-datasheet-grid#272] [Q&A]
I'm not able to have more control over the Date column, such as formatting the date. Hence, when I tried to use react-datepicker, the calendar pop-up seems to be contained within the cell. 
![20230301_181950-min](https://user-images.githubusercontent.com/55271095/222145300-4d2fcfd4-70eb-4061-ba7c-5c3fda1d4135.jpg)

**Comment by nick-keller** [comment]
You can either make the row higher, or better: only show the controls when the cell is active in an absolute positioned div.

**Reply by inaminute** [reply]
Hey, thanks for responding. 
I did try to position this with absolute, but still not able to get it working
![20230301_232538](https://user-images.githubusercontent.com/55271095/222222816-7138b0b3-e5d5-40df-8082-980aa7f5dec4.jpg)

**Comment by inaminute** [comment]
I was finally able to fix this issue through the use of portals.

**Comment by Dee66** [comment]
Any way you can expand on that (portal) a bit? Having real trouble formatting the date.

**Reply by inaminute** [reply]
In my case, the Date picker wasn't popping out of the cell, so I referred to [react portals](https://react.dev/reference/react-dom/createPortal#rendering-a-modal-dialog-with-a-portal) was able to render the React date picker over the page. [example](https://reactdatepicker.com/#example-portal-version)
